docs/supported.md

```diff
@@ -139,9 +139,9 @@ export async function getStaticPaths() {
 
 ### Revalidating Data and `next/cache`
 
-Revalidation and `next/cache` are not supported on Cloudflare Pages due to how they are implemented in Next.js. They appear to depend on Vercel infrastructure to deal with something referred to as the [suspense cache](https://github.com/vercel/next.js/blob/canary/packages/next/src/server/lib/incremental-cache/fetch-cache.ts#L38). This is used by the default fetch cache, which forms part of the incremental cache for revalidation.
+Revalidation and `next/cache` are not supported on Cloudflare Pages. This is used by the default `fetch` cache, which forms part of the incremental cache for revalidating data inside the App Router. Revalidating tags and data for an entire path also uses `next/cache`.
 
-Revalidating tags and paths goes through this functionality and appears to be what `next/cache` is built around. Because of this, it is highly unlikely that the default `next/cache` functionality will be supported on Cloudflare Pages.
+Because of this, `next/cache` functionality currently is not supported on Cloudflare Pages. The Next.js cache does work when self-hosting, and can be optionally provided a [custom cache handler](https://nextjs.org/docs/app/api-reference/next-config-js/incrementalCacheHandlerPath). It's possible this could use Cloudflare KV or Durable Objects in the future.
```

Comment:
The previous paragraph mentions that `next/cache` is not supported on Cloudflare Pages, and here in this paragraph we say "Because of this, `next/cache` functionality currently is not supported on Cloudflare Pages."
That looks a bit incorrect and redundant to me. 🤔 

```suggestion
The Next.js cache does however work when self-hosting by optionally providing a [custom cache handler](https://nextjs.org/docs/app/api-reference/next-config-js/incrementalCacheHandlerPath). It's possible this could use Cloudflare KV or Durable Objects in the future.
```